ext/soap/php_encoding.c

```diff
@@ -308,7 +308,6 @@ static zend_bool soap_check_zval_ref(zval *data, xmlNodePtr node) {
 			if (node_ptr == node) {
 				return 0;
 			}
-			xmlNodeSetName(node, node_ptr->name);
 			xmlSetNs(node, node_ptr->ns);
```

Comment:
If the name is no longer set, should the NS set also be dropped?

Reply:
@nikic I believe this is probably also problematic, but I will need a test case before being comfortable with removing it.

Reply:
I tried to come up with a case where there'd be different namespaces, but had trouble. Comparing test runs with and without the namespace copied didn't make a difference.